website/docs/r/spring_cloud_service.html.markdown

```diff
@@ -0,0 +1,132 @@
+subcategory: "App Platform"
```

Comment:
@jackofallops just curious about this rule, because in my previous comments, Tom told me to remove the first line.  https://github.com/terraform-providers/terraform-provider-azurerm/pull/4928#discussion_r394643438
so in my future PRs, should I add the first line `---` ?